app/views/miq_ae_customization/_dialog_sample.html.haml

```diff
@@ -82,19 +82,32 @@
                             - if field.type == "DialogFieldDropDownList"
                               - if field.required
                                 - if field.default_value.nil?
-                                  = select_tag('field.id',
-                                    options_for_select(val.collect(&:reverse), field.default_value))
+                                  - if field.multi_value?
```

Comment:
a lot of duplicated code, you should be able do one condition where you store {} or {:multiple => true} and always pass it